src/Microsoft.AspNet.Authentication.Cookies/CookieAuthenticationOptions.cs

```diff
@@ -152,5 +153,13 @@ public string CookieName
         /// to the client. This can be used to mitigate potential problems with very large identities.
         /// </summary>
         public IAuthenticationSessionStore SessionStore { get; set; }
+
+        CookieAuthenticationOptions IOptions<CookieAuthenticationOptions>.Value
```

Comment:
Cookies is the only one that NEEDS this, the social auths mostly will only be 1 per app, so they don't need the extra overload or gimmick for the options